### DISCUSSION TITLE
How to change Lock-Screen Wallpaper on Budgie Desktop

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#368] [Q&A] [answered]
Hi there.
How can I change the Lock-Screen Wallpaper on my budgie desktop?
I do NOT mean the Login Screen. I mean the Wallpaper when locking the screen in a running session.

I changed the normal wallpaper and also the Login-Screen Wallpaper (lightdm-settings), but the Lock screen wallpaper keeps staying to the green original wallpaper.

**Comment by AlphaElwedritsch** [comment]
Did a little bit more research

The default value is always used. Not the manually set file.

If I rename the user picture with the default file name it is displayed. This proves that the default value is always used.

Is this a bug that the user value has no effect?


![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/e2281b1d-7b30-46f0-ab24-12fbd88faef3)

**Comment by geckolinux** [comment]
Yep, Gnome and Budgie are very stubborn about changing the lockscreen wallpaper. Try changing the image path and name in `/usr/share/glib-2.0/schemas/99_spirallinux-backgrounds.gschema.override` and then run `sudo glib-compile-schemas /usr/share/glib-2.0/schemas/` .

**Reply by AlphaElwedritsch** [reply]
understood. Thanks
but I don't understand that exactly
how should I change the image path?
create the folder?
`touch /usr/share/glib-2.0/schemas/99_spirallinux-backgrounds.gschema.override`
then: `sudo glib-compile-schemas /usr/share/glib-2.0/schemas/`
and then put my wallpaper in there?

**Reply by geckolinux** [reply, marked as answer]
Hi there, `/usr/share/glib-2.0/schemas/99_spirallinux-backgrounds.gschema.override` is already present in your system:

```
[org.gnome.desktop.background:Budgie]
picture-uri = 'file:///usr/share/wallpapers/Zwopper-Green-Dew-CC-BY-SA-30-2560x1600.png'

[org.gnome.desktop.screensaver:Budgie]
picture-uri = 'file:///usr/share/wallpapers/Zwopper-Green-Dew-CC-BY-SA-30-2560x1600.png'
lock-delay = 3600

**Reply by AlphaElwedritsch** [reply]
ah, now I understand it. I just got in my own way ;)

worked. Many thanks
